programs/ssl/ssl_mail_client.c

```diff
@@ -367,6 +370,14 @@ int main( int argc, char *argv[] )
     char *p, *q;
     const int *list;
 
+#if defined(MBEDTLS_PLATFORM_C)
+    if( ( ret = mbedtls_platform_setup( &platform_ctx ) ) != 0 )
+    {
+        mbedtls_printf( " failed\n  !  mbedtls_platform_setup returned %d\n\n", -ret );
```

Comment:
We're displaying a negative return value here, yet elsewhere, are displaying it without the negative. We should be self-consistent within the same program.